packages/web/docs/src/content/gateway/other-features/security/aws-sigv4.mdx

```diff
@@ -0,0 +1,103 @@
+---
+searchable: false
+---
+
+# AWS Signature Version 4 (SigV4)
+
+Hive Gateway allows you to sign subgraph requests with AWS Signature Version 4 (SigV4) for secure
```

Comment:
Add link to AWS's page for Sig v4: https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html

Reply:
Done!